ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml

```diff
@@ -252,6 +252,62 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )manifest,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I'm not 100% sure but this might need to be false when using run_if_changed

Reply:
Don't know, is there a documentation somewhere where to check it?